package/MDAnalysis/coordinates/DCD.py

```diff
@@ -200,6 +200,9 @@ def _read_frame(self, i):
 
     def _read_next_timestep(self, ts=None):
         """copy next frame into timestep"""
+        if ts:
+            warnings.warn("ts argument to _read_next_timestep is deprecated as of 2.7.0 and will be removed in 3.0.0, see #3928")
```

Comment:
Also these need to throw a deprecation warning rather than a (I believe by default) userwarning

Reply:
Yes very right.

Reply:
This is marked as resolved but looks like the change hasn't been pushed? I'm going to unresolve if that's ok.

Reply:
Sorry hallucinating on my end.